pyaedt/generic/compliance.py

```diff
@@ -340,6 +676,10 @@ def _add_lna_violations(self, report, pdf_report, image_name, local_config):
                 "Test Result",
             ]
         ]
+        if not trace_data:  # pragma: no cover
+            msg = "Failed to get Solution Data. Check if the design is solved or the report data are correct."
```

Comment:
```suggestion
            msg = "Failed to get solution data. Check if the design is solved or if the report data is correct."
```
US English assumes "data" is a collection of pieces of information and treats it as a singular noun.